tests/queries/0_stateless/01413_modify_and_rename_in_sigle_alter.sql

```diff
@@ -0,0 +1,16 @@
+DROP TABLE IF EXISTS test_modify_and_rename;
+
+CREATE TABLE test_modify_and_rename(modify_and_rename_column Nullable(String), rename_and_modify_column Nullable(String)) ENGINE = MergeTree() PARTITION BY tuple() ORDER BY tuple();
```

Comment:
Just add some data into this table and probably alter will fail.

Reply:
Similar query also have this problem, I'll try to fixes it.

Other query for example:
```sql
ALTER TABLE test_modify_and_rename
    RENAME COLUMN modify_and_rename_column TO modify_and_rename_column_new,
    DROP COLUMN modify_and_rename_column_new
```